resources/views/editclass.blade.php

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Update Class</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <style>
+    * {
+      font-family: "Lato", sans-serif;
+    }
+  </style>
+</head>
+
+<body>
+  <main>
+    <div class="container my-5">
+      <div class="bg-light p-5 rounded">
+        <h2 class="fw-bold fs-2 mb-5 pb-2">Update Class</h2>
+        <form action="" method="POST" class="px-md-5"  enctype="multipart/form-data">
+          @csrf
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Class Title:</label>
+            <div class="col-md-10">
+              <input type="text" placeholder="Calculus" class="form-control py-2"  name="title"   value="{{$class['classTitle']}}"/>
+            </div>
+          </div>
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Capacity:</label>
+            <div class="col-md-10">
+              <input type="number" step="0.1" placeholder="Enter cap" class="form-control py-2"  name="capacity"    value="{{$class->capacity}}"/>
+            </div>
+          </div>
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Price:</label>
+            <div class="col-md-10">
+              <input type="number" step="0.001" placeholder="Enter price" class="form-control py-2"  name="price"    value="{{$class->price}}"/>
+            </div>
+          </div>
+          <div class="form-group mb-3 row">
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">Beginning Time:</label>
+            <div class="col-md-10">
+            <label for="" class="form-label"><input type="time" placeholder="Enter begTime" class="form-control" name="begTime"   value="{{ $class['begTime']}}" ></label> 
```

Comment:
try using Carbon if you want to format date,, but actually you don't need to format the time here,,

take this example if value is 4:30 AM, all you need to put in input value `value=4:30`

if 8:30 PM ,, value `value=20:30`